packages/ihe-gateway/server/Channels/Default Group/Test Channel.xml

```diff
@@ -1,224 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?><channel version="4.4.2">
-  <id>a3f9140d-1f4c-41b8-a414-d1a793c6054b</id>
-  <nextMetaDataId>2</nextMetaDataId>
-  <name>Test Channel</name>
```

Comment:
why is this being removed?

Reply:
We created 2 other tester channels, a XML and a JSON one. We remove this before (as we did the other inbound processors that are not being used anymore).

But because of the way the full backup and separated channel files interact w/ each other, those are rising from the dead and being a nuisance. Prob my last attempt 😄